api.raml

```diff
@@ -320,8 +330,7 @@ schemas:
       responses:
         200:
           description:
```

Comment:
it's in, https://github.com/silinternational/idp-id-broker/commit/fc791e5a55b97d62a70d1178b079d83b346fcee8.  Did your plugin flag that as a syntax error?  Mine didn't...